include/mbedtls/platform.h

```diff
@@ -70,7 +70,9 @@ extern "C" {
 #if !defined(MBEDTLS_PLATFORM_NO_STD_FUNCTIONS)
 #include <stdio.h>
 #include <stdlib.h>
+#if defined(MBEDTLS_HAVE_TIME)
```

Comment:
Thanks for noticing and fixing this. Would you mind raising it as a separate PR, with an appropriate changelog entry? That PR will need to be backported to long-time support branches.

Reply:
I think this has been raised as https://github.com/ARMmbed/mbedtls/pull/3624